template.rb

```diff
@@ -135,6 +135,111 @@
 HEREDOC
 create_file 'bin/update', BIN_UPDATE, force: true
 
+BIN_BUILD = <<~HEREDOC.strip_heredoc
+  #!/usr/bin/env bash
+
+  set -o errexit
+  set -o pipefail
+  set -o nounset
+
+  echo "=========== setting env variables ==========="
+  export RAILS_ENV='test'
+  export BUNDLE_APP_CONFIG='.bundle/ci-build'
+
+  echo "=========== install bundler ==========="
+  bundler_version=`tail -n 1 Gemfile.lock | xargs`
+  time gem install bundler:$bundler_version --no-document
+
+  echo "=========== bundle install ==========="
+  time bundle install
+
+  echo "=========== bundle audit ==========="
+  time bundle exec bundle-audit check --update
+
+  echo "=========== zeitwerk check ==========="
+  time bundle exec rails zeitwerk:check
+
+  echo "=========== brakeman ==========="
+  time bundle exec brakeman
+
+  echo "=========== rubocop  ==========="
+  time bundle exec rubocop .
```

Comment:
Why is the dot needed?

```suggestion
  time bundle exec rubocop --format simple
```
This does not output dots but does describe the errors.